App/SB/components/ThreadListCard/index.js

```diff
@@ -9,14 +9,22 @@ import Avatar from '../../../Components/Avatar'
 import styles, { cardImageContainerStyle, cardImageStyle } from './statics/styles'
 
 const ThreadCard = props => {
-  const { name, latestPeerId, updated, userCount, photos, onPress } = props
-  const lastUpdatedTime = moment(updated).fromNow()
-  const commentsCount = photos.length
-  let uri = latestPeerId ? 'https://cafe.us-east-1.textile.io/ipns/' + latestPeerId + '/avatar' : undefined
+  const { name, updated, userCount, photos, onPress } = props
 
-  // ensure we have the user's latest avatar even if the cafe is still caching
-  if (props.profile && props.profile.id && props.profile.id === latestPeerId) {
-    uri = 'https://cafe.us-east-1.textile.io' + props.profile.avatar_id
+  let title = name
+  if (title.length > 17) {
+    title = title.substring(0, 32)
+    title = title.substring(0, title.lastIndexOf(' '))
+    while (title[title.length - 1] === '.' || title[title.length - 1] === ' ') {
+      title = title.slice(0, -1)
+    }
+    title += '...'
```

Comment:
Surprised if there is no support for automatically formatting text like this when the test is longer than the view is wide. It exists in iOS for sure, probably Android too.

Reply:
https://facebook.github.io/react-native/docs/text#ellipsizemode

Reply:
cool, already added